DSA-Campaign-Uplift-Estimation/src/main/webapp/Create/create-script.js

```diff
@@ -620,7 +620,38 @@ function determineValidity() {
       return false;
     }
   }
-  return true;
+
+  return checkDateValidity();
+}
+
+function checkDateValidity() {
```

Comment:
Why not convert both into Date objects and compare?

Reply:
Added